codegen/command.go

```diff
@@ -33,5 +33,6 @@ func Command() codegen.Command {
 		},
 		RenderProtos: true,
 		Groups:       groups,
+		CustomGoArgs: []string{"Mgithub.com/solo-io/solo-kit/api/external/envoy/api/v2/discovery.proto=github.com/envoyproxy/go-control-plane/envoy/api/v2"},
```

Comment:
solo-kit had this [hard-coded](https://github.com/solo-io/solo-kit/blob/bb65ee2700a025bd9faa00555318a1fa4cd192fd/pkg/code-generator/collector/collector.go#L252) but I've updated skv2 to be able to take this as an arg.